src/chainlit/frontend/src/hooks/clearChat.ts

```diff
@@ -1,20 +1,23 @@
-import { useRecoilValue, useSetRecoilState } from 'recoil';
+import { useRecoilValue, useResetRecoilState, useSetRecoilState } from 'recoil';
 
 import { actionState } from 'state/action';
 import { messagesState, sessionState, tokenCountState } from 'state/chat';
 import { elementState, sideViewState } from 'state/element';
+import { sessionIdState } from 'state/user';
 
 export default function useClearChat() {
   const setMessages = useSetRecoilState(messagesState);
   const setElements = useSetRecoilState(elementState);
   const setActions = useSetRecoilState(actionState);
   const setSideView = useSetRecoilState(sideViewState);
   const setTokenCount = useSetRecoilState(tokenCountState);
+  const resetSessionId = useResetRecoilState(sessionIdState);
   const session = useRecoilValue(sessionState);
 
   return () => {
     session?.socket.emit('clear_session');
     session?.socket.disconnect();
+    resetSessionId();
```

Comment:
does reset clear the variable or create a new uuid?

Reply:
I though it would create a new uuid but that was dumb, it's obviously reusing the same uuid, looking for a workaround right now. :)

Reply:
@willydouhard should be better now